tests/pages/test_product_navigation_heuristics.py

```diff
@@ -125,3 +126,16 @@ async def test_crawl_nofollow_links():
         request_url, response, navigation, page_params
     )
     assert [req.url for req in page.subCategories] == ["https://example.com/can-follow"]
+
+
+def test_deprecated_page_objects():
+    with pytest.warns(DeprecationWarning, match="page_objects"):
+        from zyte_spider_templates.page_objects import (  # noqa: F401
+            HeuristicsProductNavigationPage,
+        )
+
+    # We cannot test the warning again because duplicate warnings are ignored,
+    # but we still want to ensure that we can import the class.
+    from zyte_spider_templates.page_objects.product_navigation_heuristics import (  # noqa: F401, F811
+        HeuristicsProductNavigationPage,
+    )
```

Comment:
If it's the successful import we want to check, what do you think about moving this 2nd check to another test function? so that we can assert if a warning was generated or not.

Reply:
I tried that, does not help :cry:

Reply:
Fair enough :D 

I'm not sure if https://docs.pytest.org/en/7.1.x/reference/reference.html#pytest.deprecated_call would also help here.

Reply:
Also tried that :sweat:

I think it is Python itself that ignores duplicate warnings. There might be a way to do it, but I could not find it in a reasonable time, and I am not sure if it is worth the effort here.